### PR TITLE
ci: disable flaky Puppeteer step

### PR DESCRIPTION
**Pull request by valerybugakov**

### CODE REVIEW

**Review by umpox — APPROVED** [review]

File: enterprise/dev/ci/internal/ci/operations.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func CoreTestOperations(changedFiles ChangedFiles, buildOptions bk.BuildOptions)
 
 	// Add additional steps
 	if changedFiles.affectsClient() {
-		operations = append(operations, clientIntegrationTests, clientChromaticTests(false))
+		operations = append(operations /* clientIntegrationTests, */, clientChromaticTests(false))
```
Comment (pdubroy) [inline enterprise/dev/ci/internal/ci/operations.go:76]: Don't forget to open a bug to re-enable this. You can use the flaky test template linked from [here](https://docs.sourcegraph.com/dev/background-information/testing_principles#flaky-tests).

Then you can add a comment here with a link to the bug.

File: enterprise/dev/ci/internal/ci/operations.go [hunk 2]
```diff
@@ -163,7 +163,7 @@ func addBrowserExt(pipeline *bk.Pipeline) {
 		bk.Cmd("dev/ci/codecov.sh -c -F typescript -F unit"))
 }
 
-func clientIntegrationTests(pipeline *bk.Pipeline) {
+func clientIntegrationTests(pipeline *bk.Pipeline) { //nolint // Step is temporarily disabled
```
Comment (pdubroy) [inline enterprise/dev/ci/internal/ci/operations.go:166]: Please link the issue here.

File: enterprise/dev/ci/internal/ci/pipeline.go [hunk 3]
```diff
@@ -163,7 +163,7 @@ func GeneratePipeline(c Config) (*bk.Pipeline, error) {
 			appendOps(addBackendIntegrationTests)
 		}
 		if c.RunType.Is(MainDryRun, MainBranch) {
-			appendOps(clientIntegrationTests, clientChromaticTests(c.RunType.Is(MainBranch)))
+			appendOps( /* clientIntegrationTests, */ clientChromaticTests(c.RunType.Is(MainBranch)))
```
Comment (pdubroy) [inline enterprise/dev/ci/internal/ci/pipeline.go:166]: ...here.

File: enterprise/dev/ci/internal/ci/web-integration-files.go [hunk 4]
```diff
@@ -1,3 +1,4 @@
+//nolint
```
Comment (pdubroy) [inline enterprise/dev/ci/internal/ci/web-integration-files.go:1]: ...and here. Also, can you add details about why you need nolint here, so someone has enough context to remove it in the future? Also, is it possible to just disable the particular thing that's failing?

**Review by pdubroy — APPROVED** [review]
Thanks for looking into this!

**Comment by valerybugakov** [comment]
It seems like it was the issue with the Puppeteer API. The pipeline step looks stable.

**Pull request closed without merging**